docs/README.md

```diff
@@ -8,7 +8,7 @@ To build the documentation, follow the steps below.
 
 Install [Doxygen](http://www.doxygen.nl/manual/install.html).
 
-Install sphinx, breathe and the theme using the `requirements.txt` file in `docs/`:
+Install `breathe` and the sphinx theme using the `requirements.txt` file in `docs/`:
```

Comment:
Don't you also need to `pip install sphinx` if you are doing this locally?

Reply:
Having `sphinx` as an explicit dependency in requirements breaks things - even doing `pip install sphinx` breaks stuff. Running `pip install sphinx_rtd_theme` [will install Sphinx](https://github.com/readthedocs/sphinx_rtd_theme/blob/master/setup.py#L118) and won't do the same dependency graph traversal that's broken.